api/pom.xml

```diff
@@ -18,32 +18,33 @@
 -->
 
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    
     <parent>
-      <groupId>org.eclipse.ee4j</groupId>
-      <artifactId>project</artifactId>
-      <version>1.0.1</version>
-      <relativePath/>
+        <groupId>org.eclipse.ee4j</groupId>
+        <artifactId>project</artifactId>
+        <version>1.0.2</version>
+        <relativePath/>
     </parent>
 
     <modelVersion>4.0.0</modelVersion>
     <groupId>jakarta.el</groupId>
     <artifactId>jakarta.el-api</artifactId>
     <packaging>jar</packaging>
-    <version>3.0.1-b07-SNAPSHOT</version>
+    <version>3.0.1-SNAPSHOT</version>
     <name>Expression Language 3.0 API</name>
 
     <properties>
         <!-- Make sure the two versions are in sync with the maven version -->
         <spec.version>3.0</spec.version>
         <bundle.version>3.0.0</bundle.version>
-        <extensionName>javax.el</extensionName>
+        <extensionName>jakarta.el</extensionName>
```

Comment:
Extension name refers to the package name and should still be javax.el

Reply:
Agreed. I'll put together a PR to fix it in master and add the same commit to this PR.